app/lib/dartdoc/dartdoc_runner.dart

```diff
@@ -508,6 +508,9 @@ bool _isKnownFailurePattern(String output) {
       output.contains('Dartdoc.logAnalysisErrors')) {
     return true;
   }
+  if (output.contains('fatal error: unable to locate the input directory at')) {
```

Comment:
Add a comment for each of these explaining why we care about this string, and what kind of error we think it is.